kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -27,7 +30,10 @@ function showHomePage(toRoute) {
 }
 
 export default [
-  ...planRoutes,
+  ...examRoutes,
+  ...lessonsRoutes,
+  ...learnersRoutes,
+  ...groupsRoutes,
   ...reportRoutes,
```

Comment:
I am planning on keep these report routes for reports that are in immersive pages that dont show a main nav item active and are common to different models (lessons, learners, quizzes, groups)

Reply:
I must admit, I'm not quite sure I understand - when I see some examples, perhaps it will be clearer!